code/modules/clothing/suits/ego_gear/waw.dm

```diff
@@ -164,3 +164,11 @@
 	armor = list(RED_DAMAGE = 40, WHITE_DAMAGE = 10, BLACK_DAMAGE = 60, PALE_DAMAGE = 30)
 	attribute_requirements = list(
 							FORTITUDE_ATTRIBUTE = 60)
+
+/obj/item/clothing/suit/armor/ego_gear/green_stem
+	name = "green stem"
```

Comment:
We already have green stem armor made...

Reply:
Oh i see it now. It was put as stem ill use that.
I changed the requirements to the original prudence Attribute 60. It will make someone who focuses on temperance need to focus on prudence or vise versa to use both weapons. 
Does 4/1/7/2 good? Since snow whites apple is immune to black damage id like if the armor also inherited a bit of her resistance.

Reply:
Go for it.